src/modules/price-router/Extensions/Curve/Curve2PoolExtension.sol

```diff
@@ -53,6 +53,11 @@ contract Curve2PoolExtension is Extension {
      */
     error Curve2PoolExtension_POOL_NOT_SUPPORTED();
 
+    /**
+     * @notice While getting the virtual price from the pool, the virtual price was outside of normal safe bounds.
+     */
+    error Curve2PoolExtension_BOUNDS_EXCEEDED();
```

Comment:
Do we want to add more detail to the error? As in the `bounds`, and the failing `value`?

Reply:
nah I dont think it is needed. This is an error that would normally only occur in a scenario where we are being attacked. Users are very unlikely to see this error, so providing extra context to it, isn't really helping the people we want to help.

Reply:
Makes sense to me!